### ISSUE TITLE
Add support for `cargo wix /path/to/Cargo.toml`

### ISSUE THREAD

**Issue by slonopotamus** [volks73/cargo-wix#86]
Subj, the same way as `cargo wix init /path/to/Cargo.toml` works.

This will

1. Decouple MSI generation from working directory
2. Allow to fix running integration tests in parallel. This currently does not work because tests change working directory back and forth, but since all integration tests run in a single process, chaos arises.

**Comment by volks73** [comment]
This sounds similar enough to #70 to be a duplicate. Can you take a look and indicate how this would be different?

>Allow to fix running integration tests in parallel. This currently does not work because tests change working directory back and forth, but since all integration tests run in a single process, chaos arises.

Well, this was done intentionally for two reasons: (1) initially the subcommand had no way of executing outside of the cwd for a Rust/cargo project, i.e. there was no way to submit a path to a Cargo.toml file, so the tests had to change the cwd and (2) I wanted to as closely mimic the majority use case for a user in the tests. I believe most of the time, the `cargo wix` subcommand would be executed from the Rust/cargo's project root. 

I recognize changing the cwd causes issues and is sometimes bad practice. It also requires serial testing. Now that for the `init` command at least, a path can be specified, the tests should probably be updated.

**Comment by slonopotamus** [comment]
Okay, duplicate of #70.

WRT tests - cargo-wix README talks about running tests in serial as though you're not exactly sure what's the cause of the problem, so I wanted to make it clear that issues are caused by changing of working directory in tests.
